storage/include/blockchain/merkle_tree_db_adapter.h

```diff
@@ -41,33 +42,61 @@ class DBKeyManipulator : public DBKeyManipulatorBase {
   static sparse_merkle::Hash extractHashFromLeafKey(const concordUtils::Key &key);
 };
 
+// The DBAdapter class provides facilities for managing a key/value blockchain on top of a key/value store.
```

Comment:
Agreed, it can be made clearer. I will do so when I provide a switch to select direct KV or merkle storage as I will possibly change interfaces and introduce common ones.